usr/sbin/pkcsstats/pkcsstats.c

```diff
@@ -837,12 +918,17 @@ int main(int argc, char **argv)
         goto done;
     }
 
+    if (json)
+        printf("{\n\t\"users\": [\n");
```

Comment:
Why a struct containing only one property?  We could simply print a json array here?

Reply:
Thats the root of the JSON tree, isn't the root required to be a struct ?

Reply:
and I wanted to call that think 'users'. So just having an array would not tell what the array is for.

Reply:
> isn't the root required to be a struct ?

No, any value is fine according to new ECMA standard.  So specifically an array would be okay.

Reply:
> I wanted to call that think 'users'.

Fine with me.  If this is supposed to be consumed by other tools you anyway need to clearly describe this API and consider it an API.  I am just not sure which other top-level key you might want to add in the future...

Reply:
The `zcryptstats` tool of s390-tools also provides JSON output, and it adds system information and a timestamp into the top level node. 
So maybe we should do this here, too ? Opinions?

```
static int json_print_header(void)
{
	char timestamp[64];
	struct utsname un;
	struct tm *tm;
	time_t t;

	time(&t);
	tm = localtime(&t);
	strftime(timestamp, sizeof(timestamp), "%x", tm);

	if (uname(&un) != 0)
		return -errno;

	printf("{\"zcryptstats\": {\n");
	printf("\t\"host\": {\n");
	printf("\t\t\"nodename\": \"%s\",\n", un.nodename);
	printf("\t\t\"sysname\": \"%s\",\n", un.sysname);
	printf("\t\t\"release\": \"%s\",\n", un.release);
	printf("\t\t\"machine\": \"%s\",\n", un.machine);
	printf("\t\t\"date\": \"%s\",\n", timestamp);
	printf("\t\t\"statistics\": [\n");

	return 0;
}
```

Reply:
Might be a good idea to already include it.  Otherwise people will be forced to write some wrapper exporters to modify the JSON to add this data.

But please no `%x` for `strftime`.  Locale-awareness might be a good idea for user-readable things, but is a horror for something I want to process further via some code.  Personally I would go for `%F` which produces some ISO-standard date format.

Reply:
done